### ISSUE TITLE
License key not remembered

### ISSUE THREAD

**Issue by dvdheiden** [laravel-shift/workbench-issues#81]
After the update to 1.0.2 I'm having issues with workbench not remembering my license key. Hopefully, this is not just on my laptop.

**Desktop:**
 - OS: Mac
 - OS Version: 12.5
 - App Version: 1.0.2

**Describe the bug**

After every start, I have to provide the license key again. But then the activation complains about having too many activated licenses. So then I have to disable a device and provide the license key. Every time 😄 

**To Reproduce**
1. Start the laptop
2. Open workbench
3. Activate the license
4. Shutdown laptop
5. Start the laptop
6. Open workbench
7. No license key activated

**Comment by jasonmccreary** [comment]
Was everything working prior to v1.0.2?

**Comment by dvdheiden** [comment]
Yes I think it was. Should I try a reinstall or is there something else I could try?

**Comment by jasonmccreary** [comment]
v1.0.2 did contain an update to the license key renewal, so it's possible there's a bug. However, I'm not able to replicate it and haven't heard any other reports.

Could you email your license key to support@laravelshift.com so I can take a closer look at your specific details.

**Comment by dvdheiden** [comment]
Of course, no problem.

**Comment by dvdheiden** [comment]
Closing this issue as the desktop app seems to be discontinued
